README.md

```diff
@@ -8,7 +8,70 @@ assisted close reading tool
 
 [https://github.com/ADAH-EviDENce/evidence-gui](https://github.com/ADAH-EviDENce/evidence-gui)
 
+## Running a demo on Windows
 
+Prerequisites:
+- Docker desktop
+
+[Download](https://github.com/ADAH-EviDENce/WorkingTitleCloseReader/archive/master.zip) a copy of WorkingTitleCloseReader archive and extract its contents on your machine.
+
+Open a Powershell prompt:
+Change your current working directory to where you extracted the files. For instance:
+
+```shell
+cd C:\Users\JohnDoe\Downloads\WorkingTitleCloseReader-master\WorkingTitleCloseReader-master
+```
+
+The demo can be started with the following command:
+
+```shell
+docker-compose --env-file getuigenverhalen.env up --build
```

Comment:
Is docker-compose coming with docker-desktop?

Reply:
Yep, see https://docs.docker.com/desktop/